Configuration/Applications/python/Options.py

```diff
@@ -428,6 +428,12 @@
                             type=str,
                             dest="heap_profile")
 
+expertSettings.add_argument("--maxmem_profile",
+                            help="add the PerfTools/MaxMemoryPreload monitor",
+                            default=None,
+                            type=str,
```

Comment:
Something like
```suggestion
                            action="store_true",
                            default=False,
```
should fix it.